### ISSUE TITLE
Scrolling doesn't work after load if no mouse action performed

### ISSUE THREAD

**Issue by JanJakes** [DataTables/FixedColumns#36]
This issue is visible even in the official examples. Steps to reproduce it:

1. Go to https://www.datatables.net/extensions/fixedcolumns/.
2. Place mouse over the example table's body or left (fixed) column.
3. Refresh the page but do not move the mouse after load.
4. Scroll with the mouse wheel. Now left column will be out of sync with the rest of the table - after any mouse move or event click they get in sync.

Tried in Chrome & Safari on Mac OS.

**Comment by DataTables** [comment]
Thanks for pointing this out. It appears to be caused by the use of `mouseover` [here](https://github.com/DataTables/FixedColumns/blob/master/js/dataTables.fixedColumns.js#L472). I'll look into this when I can (bit of a back log at the moment!).

**Comment by JanJakes** [comment]
Yes, similarly [here](https://github.com/DataTables/FixedColumns/blob/master/js/dataTables.fixedColumns.js#L494) and [here](https://github.com/DataTables/FixedColumns/blob/master/js/dataTables.fixedColumns.js#L517).

A quick hotfix in an application would be to call the event manually after page load:
```
$('.dataTables_scrollBody').trigger('mouseover');
```

But that would sync scrolling only from the table body to the left (or right) fixed columns. If I want to trigger also a sync from the left column to the body:
```
$('.DTFC_LeftBodyLiner').trigger('mouseover');
```
Then the first hotfix (from left column to the body) stops working. So I'm able to workaround this only one-way at the moment, not both.

**Comment by JanJakes** [comment]
Just digging more into it - it seems that a working solution is to set `mouseController` on the scroll event (not during the mouseover before) - would that cause any troubles? Because on window load there's no chance of getting the mouse position and figuring out which element is it over...

**Comment by DataTables** [comment]
If I recall correctly, I used mouseover rather than scroll because if you update on the scroll event, it will be updated when scrolling the main content as well as the fixed content. The result would be a never ending loop of scroll events - one triggering another (I'll need to check to confirm that!)

The fix might be to have the `mouseController` as null initially, and then set it on scroll if it is null. Keep the existing mouseover code for once the page is up and running.

**Comment by DataTables** [comment]
This was fixed by abd54c61ec710519a7b22b5f32cbef241849054f - I forgot to update this issue, sorry!
